etl/pypasar/omop/device_exposure.py

```diff
@@ -22,14 +22,212 @@ def execute(self):
             raise err
 
     def initialize(self):
-        pass
+        with self.engine.connect() as connection:
+            with connection.begin():
+                # Set schema
+                connection.execute(
+                    text(f'SET search_path TO {os.getenv("POSTGRES_OMOP_SCHEMA")}'))
+
+                # Drop stg__device_exposure view if exists
+                connection.execute(text("DROP VIEW IF EXISTS stg__device_exposure"))
+
+                # Delete device_exposure table
+                connection.execute(text("TRUNCATE TABLE device_exposure"))
 
     def process(self):
         # In batches
+        omop_schema = os.getenv("POSTGRES_OMOP_SCHEMA")
+        postop_schema = os.getenv("POSTOP_SOURCE_SCHEMA")
+        preop_schema = os.getenv("PREOP_SOURCE_SCHEMA")
```

Comment:
Please update to the existing env variables https://github.com/satish-a0/pasar/blob/main/etl/.env.example#L8-L10

Reply:
Thanks for informing. Already updated.